### PR TITLE
ENG-224 let user to upload multiple files

### PR DESCRIPTION
**Pull request by ichalagashvili**
This PR enables:
- to upload multiple files per upload

does not enable:
- uploading folders too along with single files

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ichalagashvili**

If they are not already assigned, you can assign the PR to them by writing `/assign @ichalagashvili` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/ui/common/form/RenderFileInput.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```
Comment (gidesan) [inline src/ui/common/form/RenderFileInput.js:1]: please remove it :)

File: src/locales/en.js [hunk 2]
```diff
@@ -403,6 +403,7 @@ export default {
     'fileBrowser.createTextFile': 'Create text file',
     'fileBrowser.createFolder': 'Create folder',
     'fileBrowser.uploadFile': 'Upload File',
```
Comment (gidesan) [inline src/locales/en.js:405]: is the singular entry still used?

File: src/ui/common/form/RenderFileInput.js [hunk 3]
```diff
@@ -6,31 +7,40 @@ const RenderFileInput = ({
   input: { name, onChange, onBlur },
   append, label, labelSize, placeholder, meta: { touched, error },
   help, disabled, acceptFile,
-}) => (
+}) => {
+  const genFilesArray = (files) => {
+    const filesArray = [];
+    Object.values(files).map(f => filesArray.push(f));
```
Comment (gidesan) [inline src/ui/common/form/RenderFileInput.js:13]: `push` with `map` is not a good practice. Please provide a cleaner way.

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/676#pullrequestreview-388320490" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**